image-viewer.py

```diff
@@ -14,6 +14,8 @@ def __init__(self):
         self.img_dir = default_directory
 
         self.root = Tk()
+        self.root.geometry("800x600+500+250")                  # Window Size And Window Position on screen
+        self.root.resizable(height=False,width=False)          # Adjustable Height and Width set to False
```

Comment:
Small comment: I'd say the `# Adjustable Height and Width set to False` comment doesn't provide value as that's indicated by the parameters inside the `resizable` function. I personally think it can be removed.

However, I do very much like the `# Window Size And Window Position on screen` comment as that's not obvious just from looking at `.geometry("800x600+500+250")`. I'd maybe note how that's formatted, something like `window width x window height + x position + y position`

Reply:
Yes, I appreciate your changes and I've edited the comments accordingly.